src/sass/components/burger-menu.scss

```
@@ -148,21 +146,34 @@
         transition: 0.4s;
     }
 
-    header {
-        height: 60px;
-    }
+    
 
-
-    .theme-switch {
-        margin-top: 239px;
-        margin-bottom: 20.5px;
-        margin-left: 16px;
-    }
+  
 
 body.dark-theme .menu-burger__header span,
 body.dark-theme .menu-burger__header:after,
 body.dark-theme .menu-burger__header:before {
     background: $color-background;
     
 }
-}
\ No newline at end of file
+
+.search-form {
+    margin-right: 20px;
+}
+
+}
+
+@media screen and (max-width: 321px) {
+.theme-switch {
+    display: none;
+}
+
+.search-form__icon {
+
+    display: flex;
+
+}
+
+
+}
+
```

треба пофіксати відступи і зайві лапки

```suggestion
.search-form {
    margin-right: 20px;
}

@media screen and (max-width: 321px) {
    .theme-switch {
        display: none;
    }

    .search-form__icon {
        display: flex;
    }
}

```